src/commands/commit/index.js

```diff
@@ -7,6 +7,13 @@ import withHook from './withHook'
 import withClient from './withClient'
 
 const commit = (mode: 'client' | 'hook') => {
+  // Allow to interrupt the hook without cancelling the commit
+  process.on('SIGINT', () => {
+    if (mode === 'hook') process.exit(0)
+    console.error('gitmoji-cli was interrupted')
+    process.exit(1)
+  })
```

Comment:
I would move this to the `withHook` file

https://github.com/carloscuesta/gitmoji-cli/blob/master/src/commands/commit/withHook/index.js

Reply:
I was going for this too, but we need to register the listener before prompting the user for the questions:
which mean here https://github.com/carloscuesta/gitmoji-cli/blob/master/src/commands/commit/index.js
or higher up